ui/apps/platform/src/Containers/Clusters/cluster.helpers.tsx

```diff
@@ -27,11 +27,6 @@ export const runtimeOptions = [
         tableDisplay: 'CORE BPF',
         value: 'CORE_BPF',
     },
-    {
-        label: 'eBPF Program',
-        tableDisplay: 'eBPF',
-        value: 'EBPF',
-    },
```

Comment:
@Stringy

Is this value no longer possible in response to /v1/clusters endpoint? Even if sensor is older than central?

I see `[deprecated]` in proto file.
Or does UI need to display if if in response, but not allow to select it?

Reply:
I think as you say, probably the UI needs to display it but not allow its selection. I'm currently trying to test these kinds of mismatched versions (e.g. older secured cluster with EBPF configured) which is proving to be a little tricky, but hopefully I can get a better understanding of the effect these changes have.

Reply:
It is my understanding that I should, at the very least, revert these changes to avoid the display issue, but preventing selection I'm less sure about. I'm not sure I fully understand the implications of selecting the collection method in the UI, or how this propagates, or whether it'll simply be translated in the backend to core_bpf anyway (with the changes introduced here)

I'll do some more testing tomorrow morning